core/health/api/src/main/java/org/opennms/core/health/api/SimpleHealthCheck.java

```diff
@@ -40,17 +40,39 @@
 public class SimpleHealthCheck implements HealthCheck {
 
     private final Supplier<String> descriptionSupplier;
+    private final Supplier<String> nameSupplier;
+    private final boolean isLocalCheck;
     private Response response = new Response(Status.Starting);
 
     public SimpleHealthCheck(Supplier<String> descriptionSupplier) {
+        this(descriptionSupplier, ()->"");
+    }
+
+    public SimpleHealthCheck(Supplier<String> descriptionSupplier, Supplier<String> nameSupplier) {
+        this(descriptionSupplier, nameSupplier, false);
```

Comment:
`AdapterHealthCheck` and `ListenerHealthCheck` extends this and both of them are local checks.

Reply:
Apologies, I'm lost on this one, not sure what you're trying to explain.

Reply:
Should the default value of `isLocalCheck` be true instead of false ?